CHANGELOG.rst

```diff
@@ -4,6 +4,31 @@ CHANGELOG
 This document describes changes between each past release as well as
 the version control of each dependency.
 
+26.0.0 (unreleased) 
+===================
+
+**Breaking Changes**
+
+- Do not install ``kinto_changes`` and ``kinto_signer`` plugins from PyPI.
+  The source code for these plugins has moved to this repository and will
+  continue to be maintained here. ``kinto-changes`` was copied to this repo at
+  commit 0af75a9efb6ae849ee1a6761349bfaf49dd1c488, and ``kinto-signer`` was
+  copied at commit 249db348caec02daafc4c249658b4ca2a89343bd. After they were
+  copied, linting and formatting tools were run against them and changes were
+  made for CI checks to pass. 
+
+- ``kinto_signer``'s internal package ``signer`` was renamed to ``backends``
+
+- ``kinto_changes`` and ``kinto_signer`` are now combined into one plugin
+  ``kinto_remote_settings``. Kinto config files must be changed to register the
+  plugin as such. 
+  
+- Some config files (e.g. Remote Settings server configuration, `permissions
+  metadata <https://github.com/mozilla-services/remote-settings-permissions>`_.)
+  rely on classes from ``kinto_signer`` and ``kinto_changes`` to have specific
```

Comment:
I don't have anything in mind for a setting in ``kinto_changes`` that would rely on a classname. Do you?

Maybe it could be worth noting that the settings names haven't changed (eg.  ``kinto.changes.http_post``)

Reply:
Ahh yeah, I suppose it's only `kinto_signer` that was affected by this. Which you also covered in the comment below 👍🏻